Jellyfin.Server/Program.cs

```diff
@@ -39,9 +39,14 @@ namespace Jellyfin.Server
     public static class Program
     {
         /// <summary>
-        /// The name of logging configuration file.
+        /// The name of logging configuration file containing application defaults.
         /// </summary>
-        public static readonly string LoggingConfigFile = "logging.json";
+        public static readonly string LoggingConfigFileDefault = "logging.default.json";
+
+        /// <summary>
+        /// The name of the logging configuration file containing user override settings.
+        /// </summary>
+        public static readonly string LoggingConfigFileUser = "logging.user.json";
```

Comment:
I definitely think this should just be `logging.json` because the user string implies a per-user setting.

Reply:
I'd prefer the bare name to be the original, with the `.user` overriding, but to apply the migration today i think we have to stick with this naming.

Reply:
I'd rather get it right and just push a release the next day, because you can only apply migrations once. We can always copy the original to `logging.copy.json` or something to keep a sane name for the new one rather than the current name.

Reply:
Unless we've done testing for both migrations in all scenarios, it would be better to do that before releasing as well.

Reply:
I'm not sure what the exact intentions were, from the chats I saw this method was superiour for some reason vs. the base name.

What if we had it automatically write any diffs from the default config into `.user`, and then replaced it? Though that wouldn't solve the "user has already changed settings" issue any more. We can also always do another migration in the next release to fix the names back to sensible ones.

Reply:
What if we just change the migration to use something like `logging.override.json` or `logging.system.json`.

Personally, I like the naming of `logging.default.json` because it makes it super obvious that it will be applied first then overwritten by the second file.

If we want to use `logging.default.json` for the defaults, and `logging.json` for the system-specific settings, we can also do that relatively easily. We just need to update the migration to first move the existing file to `logging.json.old` if we detect it has been unmodified.

Reply:
Yeah, that's all I was suggesting @mark-monteiro, although the file extension should always be last so I would say `logging.old.json` is better.

Reply:
Ah okay great, yeah that makes sense. I'm away from my computer until this evening so if someone else wants to make that change that would be fine with me